src/components/NotFound.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const NotFound: React.FC = () => {
```

Comment:
```suggestion
export const NotFoundPage: React.FC = () => {
```

Also change file name